cmd/terraform_native_commands.go

```diff
@@ -0,0 +1,201 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// getTerraformCommands returns an array of statically defined Terraform commands with flags
+func getTerraformCommands() []*cobra.Command {
+	// List of Terraform commands
+	return []*cobra.Command{
+		{
+			Use:   "plan",
+			Short: "Show changes required by the current configuration",
+			Long:  "Generate an execution plan, which shows what actions Terraform will take to reach the desired state of the configuration.",
+		},
+		{
+			Use:   "apply",
+			Short: "Apply changes to infrastructure",
+			Long:  "Apply the changes required to reach the desired state of the configuration. This will prompt for confirmation before making changes.",
+		},
+		{
+			Use:   "workspace",
+			Short: "Manage Terraform workspaces",
+			Long:  "Create, list, select, or delete Terraform workspaces, which allow for separate states within the same configuration.",
+		},
+		{
+			Use:   "clean",
```

Comment:
This is a custom extension in atmos, so I want to keep our custom extensions in a separate section.

Reply:
If you only care about user help output should be in seperate section. It is done. But if you meant even in code. I would like to ask why?